.github/workflows/pull-request.yml

```diff
@@ -52,6 +52,20 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  JSDocs:
+    name : "JSDocs comments and pipeline"
+    runs-on : ubuntu-latest
```

Comment:
Make this rely on `Code-Formatter` like `Code-Coverage` below. The correct functioning of this Action will probably rely on correct formatting. So we will need to make sure the ordering is maintained.